pkg/support/dump_windows_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package support
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	exectesting "k8s.io/utils/exec/testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/utils/exec"
+)
+
+type testExec struct {
+	exectesting.FakeExec
+}
+
+func (te *testExec) Command(cmd string, args ...string) exec.Cmd {
+	fakeCmd := new(exectesting.FakeCmd)
+	fakeCmd.CombinedOutputScript = append(fakeCmd.CombinedOutputScript, func() ([]byte, []byte, error) {
+		return []byte(fmt.Sprintf("%s %s", cmd, strings.Join(args, " "))), nil, nil
+	})
+	return fakeCmd
+}
+
+func TestDumpLog(t *testing.T) {
```

Comment:
why are the tests below specific to Windows?

Reply:
Ive written test cases for the functions in `dump_windows.go` in `dump_windows_test.go` file

Reply:
`DumpLog` has a different implementation on Windows and Linux, but it is defined for both platforms. Are we only testing in on Windows?

Reply:
Yeah dump_others.go also have `DumpLog` ,this will be covered in `dump_others_test.go` that is not included in this PR, Ill create another PR for that that's still in process.

Reply:
Sorry, this is still not clear to me.
Do you actually need 2 versions of the test (one for Linux and one for Windows), or can the same test validate both implementations? If there is nothing platform-specific in the test case itself, I imagine that the same test case could be used.